include/ginkgo/ginkgo.hpp

```diff
@@ -62,6 +62,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <ginkgo/core/base/precision_dispatch.hpp>
 #include <ginkgo/core/base/range.hpp>
 #include <ginkgo/core/base/range_accessors.hpp>
+#include <ginkgo/core/base/reordered.hpp>
```

Comment:
I would rename this to `scaled_reordering` ? Just `reordered` is a bit vague.

Reply:
I second that.